src/components/my-reservations/MyReservations.css

```diff
@@ -0,0 +1,106 @@
+.car-name {
+  font-size: 22px;
+  font-weight: 500;
+  padding-top: 5px;
+}
+
+.car-name,
+.description,
+.price,
+.dates {
+  padding-left: 10px;
+  padding-bottom: 10px;
+  color: rgb(38, 26, 72);
+}
+
+.description {
+  font-size: 14px;
+  color: rgb(134, 144, 161);
+  text-overflow: ellipsis;
+  height: 50px;
+}
+
+.price {
+  font-size: 19px;
+}
+
+.total_price {
+  color: rgb(50, 95, 160);
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.daily_price {
+  font-size: 16px;
+}
+
+.dates {
+  font-size: 16px;
+  font-weight: 500;
+}
+
+.card-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  gap: 30px;
+  margin: 30px;
+  padding-top: 10px;
+}
+
+.reservation-card {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(239 243 248);
+  width: 300px;
+  height: 400px;
+  overflow: scroll;
+  border-top-left-radius: 50px;
+  border-bottom-left-radius: 20px;
+  border-bottom-right-radius: 20px;
```

Comment:
Please make sure to make the card dynamic, don't fix the width and height, as it can have some unexpected behaviors: 
![image](https://user-images.githubusercontent.com/50754458/229946586-4de63a45-2921-4e25-be63-2b6327cf0408.png)
As you can see in the second card, the description overlaps the price because the height was fixed and it cant find the remaining space. Also, it's better to not have a scroll behavior for each card as it is against user experience.